packages/main/src/FileUploader.js

```diff
@@ -441,14 +441,6 @@ class FileUploader extends UI5Element {
 		return this.shadowRoot.querySelector("input[type=file]") || this.querySelector("input[type=file][data-ui5-form-support]");
 	}
 
-	/**
-	 * Determines input helper type in forms.
-	 * @private
-	 */
-	get _type() {
```

Comment:
@dimovpetar get _type() is now removed and "file" directly used in "InputElementsFormSuport#syncNativeFileInput"